src/coreclr/tests/src/tracing/eventpipe/stoponstart/stoponstart.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworkIdentifier>.NETCoreApp</TargetFrameworkIdentifier>
+    <OutputType>exe</OutputType>
+    <CLRTestKind>BuildAndRun</CLRTestKind>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <CLRTestPriority>0</CLRTestPriority>
+    <UnloadabilityIncompatible>true</UnloadabilityIncompatible>
+    <JitOptimizationSensitive>true</JitOptimizationSensitive>
```

Comment:
Does this test work with GCStress? (I've seen a lot of eventpipe tests fail with GCStress on mainly because of timing issues)

Reply:
Haven't tried.  I'll run the new tests with gcstress locally.